docs/admin/metrics/eventing-metrics-api.md

```diff
@@ -1,19 +1,16 @@
 ---
-title: "Metrics API"
-weight: 99
+title: "Knative Eventing metrics API"
+weight: 10
 type: "docs"
+aliases:
+  - /docs/eventing/metrics
 ---
 
-<br>
+Administrators can monitor the Knative Eventing control plane by looking at the metrics exposed by each Knative Eventing component.
 
-**NOTE:** The metrics API may change in the future, this serves as a snapshot of the current metrics.
+## Broker
 
-## Admin
-
-Administrators can monitor Eventing based on the metrics exposed by each Eventing component.
-Metrics are listed next.
-
-### Broker - Ingress
+### Ingress metrics
 
 Use the following metrics to debug how broker ingress performs and what events are dispacthed via the ingress component.
```

Comment:
Not your doing, but while here, could you fix the typo: dispacthed => dispatched? :)

Reply:
😂 of course, I didn't even notice it, thank you